web-services/model/src/test/java/datawave/webservice/query/model/ModelBeanTest.java

```diff
@@ -67,9 +79,12 @@ public class ModelBeanTest {
     private DatawavePrincipal principal = null;
     
     private static long TIMESTAMP = System.currentTimeMillis();
+    private static long NEWTIMESTAMP = System.currentTimeMillis() + 1000;
```

Comment:
I think the java convention is for underscores between words for static variables, e.g., `NEW_TIMESTAMP`